### PR TITLE
EWL-3572: adds the course browser tabs organism.

### PR DESCRIPTION
**Pull request by deadarm**
**Jira Ticket**

- [EWL-3572: EC | Design Browse Tabs component](https://issues.ama-assn.org/browse/EWL-3572)

## Description

Creates the course browser tabs for the Education Center section

## To Test

- [ ] Navigate to Organisms > EC > Ec Course Browser Tabs
- [ ] Verify that the Education Center Course Browser meets [requirements](https://docs.google.com/document/d/11SpEl1T1uRZ2HPg2g6i-RprZhsPXo_pt2W3xMbCnNZo/edit#):

> This section will allow for 2-5 tab groups. Each group will have 3-4 Course Cards and a Call to 
> Action link. The primary call to action will remain static throughout all of the tabs.
Fields:
Section Title (text)
Tab group (allows 2-5 items, implemented using Quick Tabs)
Group Label (text)
Course cards (allows 3-4 cards)
Primary CTA (link, default text: “Start a Course”, default url: “https://searchec.ama-assn.org”)
Secondary CTA (link field, custom text and url)
Link will change depending on which tab is currently selected, for example “Browse Patient Care Courses” should link to Patient Care courses.

## Relevant Screenshots/GIFs

N/A

## Remaining Tasks

N/A

## Additional Notes

Dynamic aspects to be included in Drupal implementation of tabs.

### CODE REVIEW

**Comment by deadarm** [comment]
This PR needs the 4-up layout from https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/138

File: styleguide/source/_patterns/02-organisms/05-ec/03-ec-tabs-course-browser/ec-tabs-course-browser.twig [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <li class="tabs-course_browser_list_item"><a href="{{ url }}" data-tab="tab-5" role="menuitem">Personal Well-being</a></li>
+  </ul>
+  <ul class="tabs-course_browser_list-cards layout layout-four-up">
+    <li class="tabs-course_browser_list-cards-item">{% include 'molecules-ec-card-course' %}</li>
+    <li class="tabs-course_browser_list-cards-item">{% include 'molecules-ec-card-course' %}</li>
+    <li class="tabs-course_browser_list-cards-item">{% include 'molecules-ec-card-course' %}</li>
+    <li class="tabs-course_browser_list-cards-item">{% include 'molecules-ec-card-course' %}</li>
+  </ul>
+  <div class="tabs-course_browser_ctas">
+    {% include 'atoms-button-link' with { 'url': "https://searchec.ama-assn.org", 'class': "tabs-course_browser_cta-primary", "content": "Start a Course" } %}
```
Comment (froboy) [inline styleguide/source/_patterns/02-organisms/05-ec/03-ec-tabs-course-browser/ec-tabs-course-browser.twig]: Please add the `button-small` class for both of these. I don't think we're using the standard buttons anywhere at the moment.

**Comment by deadarm** [comment]
Working on changes.

**Comment by deadarm** [comment]
@froboy I think that the primary CTA is supposed to show on tablet because the specs say "We should only display one CTA (Start a Course). This is required." and the Start a Course is "Primary CTA (this will default to “Start a Course” and takes you to the Search Education Page, default url: “https://searchec.ama-assn.org”)"

File: styleguide/source/_patterns/02-organisms/05-ec/03-ec-tabs-course-browser/ec-tabs-course-browser.twig (hunk 1 above)
Comment (deadarm) [inline styleguide/source/_patterns/02-organisms/05-ec/03-ec-tabs-course-browser/ec-tabs-course-browser.twig]: Adding `button-small` class.

**Comment by froboy** [comment]
@deadarm I've corrected the description in the ticket. Sorry about that conflict. 
landing_page_-_logged_out__0_3_4___5-up_
From [the wireframes](http://lleoij.axshare.com/#g=1&p=landing_page_-_logged_out__0_3_4___5-up_&view=a)

File: styleguide/source/_patterns/02-organisms/05-ec/03-ec-tabs-course-browser/_ec-tabs-course-browser.scss [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+  color: $black;
+  display: inline-block;
+  line-height: 1.3;
+  padding: 0;
+  margin-bottom: 0;
+  min-width: 190px;
+  text-align: center;
+
+  a {
+    padding: 1em 0.7em;
```
Comment (froboy) [inline styleguide/source/_patterns/02-organisms/05-ec/03-ec-tabs-course-browser/_ec-tabs-course-browser.scss]: (Somehow this didn't get posted with my original comments)

This also needs `display:block` to actually work. I think with that these are have a nice amount of space.
tabs-padding

**Comment by froboy** [comment]
Please give the dropdown items some spacing and make the links `display:block` to match the general style of the Jump Nav. I think centering the text in this case though is good.
discounts___offers_on_auto_purchases___car_rentals___ama_and_pattern_lab_-_organisms-ec-tabs-course-browser

The rest of the changes look great. Thanks for bearing with me.

File: styleguide/source/_patterns/02-organisms/05-ec/03-ec-tabs-course-browser/_ec-tabs-course-browser.scss (hunk 2 above)
Comment (deadarm) [inline styleguide/source/_patterns/02-organisms/05-ec/03-ec-tabs-course-browser/_ec-tabs-course-browser.scss]: Adding display block.

**Comment by deadarm** [comment]
Ready for re-review.

**Comment by froboy** [comment]
LGTM Thanks!

**Pull request merged**